tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -503,27 +503,44 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
     }
   }
 
-  "Document Encrypt Unmanaged" should {
+  "Document unmanaged encrypt/decrypt" should {
     "succeed for good data" in {
       val sdk = IronSdk.initialize(createDeviceContext)
       val data: Array[Byte] = List(1, 2, 3).map(_.toByte).toArray
       val maybeResult =
         Try(
-          sdk
-            .advanced()
+          sdk.advanced
             .documentEncryptUnmanaged(data, DocumentEncryptOpts.create(null, null, true, Array(), Array(), null))
         ).toEither
       val result = maybeResult.value
       result.getId.getId.length shouldBe 32
       result.getEncryptedDeks.isEmpty shouldBe false
     }
 
+    "roundtrip for single level transform for good data" in {
```

Comment:
This test name seems out of place in this SDK as we don't normally talk about transform levels in SDKs. This is unmanaged decrypt through a user, right? Do we care to draw the distinction between decrypting through a user vs a group here? If yes, should we have similar round trip test for groups?

Reply:
I made two separate roundtrip tests, for user and group. I also removed the standalone unmanaged encrypt test as it seemed unnecessary since we have the roundtrip test.